rootfs-builder/rootfs.sh

```diff
@@ -341,7 +391,12 @@ build_rootfs_distro()
 		docker_run_args+=" --runtime ${DOCKER_RUNTIME}"
 
 		if [ -z "${AGENT_SOURCE_BIN}" ] ; then
-			docker_run_args+=" --env GO_AGENT_PKG=${GO_AGENT_PKG}"
+			if [ "$RUST_AGENT" == "no" ]; then
+				docker_run_args+=" --env GO_AGENT_PKG=${GO_AGENT_PKG}"
+			else
+				docker_run_args+=" --env RUST_AGENT_PKG=${RUST_AGENT_PKG} -v ${RUST_SRC_PATH}:${RUST_SRC_PATH} --env RUST_SRC_PATH=${RUST_SRC_PATH}"
+			fi
+			docker_run_args+=" --env RUST_AGENT=${RUST_AGENT} -v ${GOPATH_LOCAL}:${GOPATH_LOCAL} --env GOPATH=${GOPATH_LOCAL}"
```

Comment:
could this `RUST_AGENT=` addition go into the `if` block above it? I presume it is not needed if just building the go version.

Reply:
We need it since I wrote the scripts to determine it is yes or no style. The scripts will be run in docker too, so we need it. Of course we can do it in yes or empty style